molo/commenting/views.py

```diff
@@ -43,3 +47,22 @@ def post_molo_comment(request, next=None, using=None):
 
     request.POST = data
     return post_comment(request, next=next, using=next)
+
+
+def view_more_article_comments(request, page_id):
+    article = get_object_or_404(ArticlePage, id=page_id)
+    qs = MoloComment.objects.for_model(ArticlePage).filter(
+        object_pk=page_id, parent__isnull=True)
+    paginator = Paginator(qs, 20)
```

Comment:
Please store the _comments per page_ in the settings file. See [this example](https://github.com/praekelt/molo.commenting/blob/develop/molo/commenting/models.py#L34-L37) for using the value from the settings file. You can use 20 as the default when nothing is specified.